extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/user/UserService.java

```diff
@@ -86,14 +87,18 @@ private void putAllUsers(Map<String, User> users, LDAPConnection ldapConnection,
             String usernameAttribute) throws GuacamoleException {
 
         try {
+	    // Set search limits
+	    LDAPSearchConstraints constraints = new LDAPSearchConstraints();
+	    constraints.setMaxResults(confService.getMaxResults());
```

Comment:
Didn't notice this before, but please use 4-space indents (no tabs). I saw that these three lines weren't quite aligned with the surrounding code, and decided to download the patch to take a closer look. The misalignment is due to the tabs (see yellow highlight in screenshot below):

![ick-tabs](https://cloud.githubusercontent.com/assets/4632905/17675615/4d2b6d90-62df-11e6-8bbc-6444a696c7d5.png)

You might have to change your editor settings to get it to stop adding tabs instead of spaces.

Other than that, I see no further issues. With the exception of the tabs, all looks good to me, even the commit history. ;)